mailpoet/tasks/phpstan/email-editor-phpstan.neon

```diff
@@ -15,6 +17,24 @@ parameters:
   ignoreErrors:
     - '#_before\(\) has no return type specified#' # skip integration test before errors
     - '#Container::get\(\) is not referenced in a parameter.#'
+    -
```

Comment:
There are many new errors to ignore and a lot of using `@phpstan-ignore-line` in this commit. I have checked the Container class code, and if we change `@param string|class-string<T> $name` to  ` @param class-string<T> $name`, we can remove almost each of them. This change needs to update the test for this class, but I think it is worth it. Can you please take a closer look?